test/moclojer/openapi_test.clj

```diff
@@ -54,38 +51,36 @@
                   "findPets"       {"status"  200
                                     "body"    (json/generate-string [{:id   0
                                                                       :name "caramelo"}])
-                                    "headers" {"Content-Type" "application/json"}}})
-
-
-        service-fn (-> {::http/routes (moclojer/make-router {::openapi/config config})}
-                       http/default-interceptors
-                       http/create-servlet
-                       ::http/service-fn)]
+                                    "headers" {"Content-Type" "application/json"}}})]
     (testing
      "findPets route"
       (is (= [{:id   0
                :name "caramelo"}]
-             (-> service-fn
+             (-> config
+                 service-fn
```

Comment:
call `(service-fn config)` on each request is like restart your http server on each request. 

this is not a good pattern. 

service-fn function creates a (potentially) stateful http server.

Reply:
done